### PR TITLE
Please Kindly Review this Branch

### PR DESCRIPTION
**Pull request by Mekuseo**
Playing the Role of a code Reviewer:
- Checking if I used the correct semantics of html tags
- Checking if I applied html and css best practices.
- Adding as many comments as possible

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,311 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="header2">
+      <a href="#">Welcome</a>
+      <nav>
+        <img class="menu-button" src="./images/Enabled.png" alt="mobile-menu" />
+      </nav>
+    </header>
```
Comment (Mekuseo) [inline index.html:22]: This is a nice header. Next time please do ensure to enclose your nav tag in a section tag.
```suggestion
<section>
    <header class="header2">
      <a href="#">Welcome</a>
      <nav>
        <img class="menu-button" src="./images/Enabled.png" alt="mobile-menu" />
      </nav>
    </header>
    </section>
```

File: index.html [hunk 2]
```diff
@@ -1,12 +1,311 @@
...
+
+    <section class="heading">
+      <div></div>
```
Comment (Mekuseo) [inline index.html:25]: This empty div tag is not necessary. You can create a space by padding or creating a margin-top

File: index.html [hunk 3]
```diff
@@ -1,12 +1,311 @@
...
+        </li>
+        <li>
+          <a href="https://angellist.com"
+            ><img src="./images/angellist.svg" alt=""
+          /></a>
+        </li>
+      </ul>
+    </section>
+
+    <main class="grid-display">
```
Comment (Mekuseo) [inline index.html:63]: This main section should equally be enclosed in a Section tag

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
+body {
+  width: 375px;
+  align-items: center;
+  background-repeat: no-repeat;
```
Comment (Mekuseo) [inline styles.css:4]: This code serves nothing here. Please kindly recheck and fix it.

**Review by Mekuseo — COMMENTED** [review]
You've done an excellent job and you are almost there.
Please kindly make changes to the highlighted sections and make your code better. 
Cheers!

**Review by Olamarx — APPROVED** [review]
Hi @Mekuseo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of GitHub review features ✔️ 
- Grammatically correct sentences ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**